Emby.Dlna/Main/DlnaEntryPoint.cs

```diff
@@ -399,11 +401,36 @@ private void DisposePlayToManager()
             }
         }
 
+        public void DisposeDevicePublisher()
+        {
+            if (_publisher != null)
+            {
+                _logger.LogInformation("Disposing SsdpDevicePublisher");
+                _publisher.Dispose();
+                _publisher = null;
+            }
+        }
+
+        /// <inheritdoc />
         public void Dispose()
         {
-            DisposeDevicePublisher();
-            DisposePlayToManager();
-            DisposeDeviceDiscovery();
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        protected virtual void Dispose(bool disposing)
```

Comment:
Add docs (just copy them from somewhere else)

Reply:
Added docs to two other `Dispose `methods in Emby.Dlna as well